var/spack/repos/builtin/packages/aom/package.py

```diff
@@ -0,0 +1,19 @@
+from spack import *
+
+class Aom(CMakePackage):
+    """Alliance for Open Media"""
+    homepage = "https://aomedia.googlesource.com/aom"
+    git      = "https://aomedia.googlesource.com/aom"
+    version('v1.0.0-errata1', commit='29d8ce4836630df5cc7ab58f1afc4836765fc212')
+    version('1.0.0', commit='d14c5bb4f336ef1842046089849dee4a301fbbf0')
```

Comment:
Which of these two versions do you want to be the default? If you run `spack info aom`, you can see how the versions currently sort.

Reply:
Prob best to use the errata as it is the one that actually works for the ffmpeg build, the other doesn't